### ISSUE TITLE
Grabbing _onBridgeCmd interfereing with other addons using JS bridge

### ISSUE THREAD

**Issue by jamesrwaugh** [mass-immersion-approach/MIA-Japanese-Add-on#92]
**Type:**
  - [x] bug
  - [ ] feature
  - [ ] enhancement
  - [ ] question

**Environment:**
  - OS: Ubuntu 21.04
  - (Working on testing fix on Windows 10)

**I'm going to open a PR to resolve the issue:**
  - [X] yes
  - [ ] no

**Description:**

(Following the contribution guidelines, opening an issue for a pull request I already submitted)

See description on the PR here: https://github.com/migaku-official/Migaku-Japanese-Addon/pull/91
Basically, I found that according to the Anki docs, Migaku should be using gui_hooks for its use of JS bridges in webviews. As it stands, other addons creating their own webviews will have issues when installed alongside Migaku.
I believe the Anki docs refer to this as "monkey patching" and is discouraged due to unpredictable side effects.
